templates/web/base/main_nav_items.html

```diff
@@ -0,0 +1,38 @@
+[%~ PROCESS navitem uri='/' label=loc('Report a problem') attrs='class="report-a-problem-btn"' ~%]
+
+[%~ IF c.user_exists ~%]
+    [%~ PROCESS navitem uri='/my' label=loc('Your account') ~%]
+[%~ ELSE ~%]
+    [%~ PROCESS navitem uri='/auth' label=loc('Sign in') ~%]
+[%~ END ~%]
+
+[%~ IF c.user_exists AND c.user.has_body_permission_to('planned_reports') ~%]
+    [%~ PROCESS navitem uri='/my/planned' label=loc('Shortlist') ~%]
+[%~ END ~%]
+
+[%~
+  IF c.user_exists AND c.user.categories.size;
+    categories = c.user.categories.join(",") | uri;
+    suffix = "?filter_category=" _ categories;
```

Comment:
I wonder if it would be clearer to pass suffix to all calls to navitem, so blank for most of them, rather than having a parent variable you have to clear.